### PR TITLE
COMEBACK-96

### PR DESCRIPTION
**Pull request by TRLevesque**
Screen Shot 2022-04-12 at 1 24 11 PM

Styling and mark up for Ramsey Apps v1.

### CODE REVIEW

**Comment by levininja** [comment]
@TRLevesque 
🚫 Please add a screenshot and a description to your main comment (on the Conversation tab).
🚫 Please change the name of the ticket to have the ticket number in it as the first thing. 
🎸 Just for future reference, it's ideal if your commits have the ticket number as the first part of the commit name, and then the rest of the commit name is description of what's in it. No need to change this review for that though.

**Review by levininja — COMMENTED** [review]
🚫  Can we name "Group 707.png" something more descriptive?

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ramsey Apps 1.0</title>
+    <link rel="apple-touch-icon" sizes="57x57" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-57x57.png">
```
Comment (levininja) [inline public/index.html:8]: 🗣️ Why do we have so many of this image in different sizes? apple-touch-icon-120x120.png, 144x144, etc.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    <link rel="apple-touch-icon" sizes="180x180" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon-16x16.png">
+    <link rel="icon" type="image/x-icon" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon.ico">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body class="bg-light brush">
+      <!--header-->
```
Comment (levininja) [inline public/index.html:25]: 🎸  Comments like this are redundant, as the tag right below it is < header > which is self-documenting.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	<header class="border-bottom mb-5">
+        <div class="container padding-4 p-4">
+            <div class="row row-cols-2">
+                <div class="col">
+                    <img src="https://cdn.ramseysolutions.net/media/rscom/logos/masthead-logo-ramsey.svg" class="" width="200" alt="Ramsey Solutions">
+                </div>
+                <div class="col d-flex">
+                    <div class="ms-auto justify-content-center align-self-center">
+                        <div class="d-flex">
+                            <a target="_blank" href="#" class="row font-bold text-decoration-none pr-4 text-blue "><span class="d-flex justify-content-center align-items-center"><span class="">Levi Hobbs </span><span class="p-1 h4 bi bi-person-circle my-auto"></span></span></a>
```
Comment (levininja) [inline public/index.html:35]: I see myself in everything you do...

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+        </div>
+    </header>
+    <!--content-->
+    <div class="container">
+        <h1 class="mb-4 font-bold text-dark-blue">
+            Hey Levi, welcome back!
+        </h1>
+        <div class="row row-cols-4">
+            <!--block1-->
```
Comment (levininja) [inline public/index.html:49]: 🎸 Personally I don't know that comments like these do a whole lot of good

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            <div class="p-2">
+                <a target="_blank" target="_blank" href="https://www.ramseysolutions.com/ramseyplus/financial-peace">
+                <div class="bg-white hover-shadow p-5 round d-flex h-100" >
+                    <img src="images/FinancialPeace-Logo-Blue-5-21.png" class="mx-auto justify-content-center align-self-center mx-auto justify-content-center align-self-center py-4 img-fluid"  alt="Finnancial Peace">
```
Comment (levininja) [inline public/index.html:53]: 🎸 JSYK img tags should have a trailing forward-slash.

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="bg-white hover-shadow p-5 round d-flex h-100" >
+                    <img src="images/SmartDollar-Logo-Color.png" class="mx-auto justify-content-center align-self-center py-4 img-fluid"  alt="Smart Dollar">
+                </div>
+            </a>
+            </div>
+             <!--block1-->
+             <div class="p-2">
+                <a target="_blank" href="https://www.ramseysolutions.com/business/entreleadership">
+                <div class="bg-white hover-shadow p-5 round d-flex h-100" >
+                    <img src="images/entreleadership-logo-blue 1.png" class="mx-auto justify-content-center align-self-center py-4 img-fluid"  alt="Entreleadership">
```
Comment (levininja) [inline public/index.html:109]: It makes me happy that all of these imgs have alt tags :-)

File: public/styles.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
+body{
+height:100vh;
```
Comment (levininja) [inline public/styles.css:2]: TIL something new. vh units. cool.

File: public/styles.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+}
+.font-bold {
+font-weight: bold;
+}
+ .brush{
+content: "";
+background-image: url(images/brush-stroke.svg);
+background-repeat: no-repeat;
+background-position: -300px 160%;
```
Comment (levininja) [inline public/styles.css:11]: 🗣️ This -300px is interesting, what's going on there out of curiosity?

File: public/styles.css [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+.text-dark-blue{
+color:rgb(23, 54, 94);
+}
+
+.text-blue{
+    color:rgb(0, 91, 152);
+    }
+
+.hover-shadow:hover{
+        box-shadow: 0 .125rem .25rem rgba(0,0,0,.075)!important;
```
Comment (levininja) [inline public/styles.css]: 🗣️ Can this important be removed without any negative effect?

File: public/styles.css [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.hover-circle:hover{
+    background-color: #e0e1e1!important;
```
Comment (levininja) [inline public/styles.css]: 🗣️ Can this important be removed without any negative effect?

**Review by levininja — CHANGES_REQUESTED** [review]
Good work getting this out there fast.

I think all the things I marked here will be easy fast things. Whenever you get done with them, add them to this PR (by doing a new commit, pushing it, then going into GH and it will have a big button showing for you to add the commit to the PR). Once it's added, let me know physically (as I will be tuning out chat most of today), and I should be able to approve and we can merge it basically instantly at that point.

Reminder about how CRs work in general: 🗣️ means initiating a discussion here, doesn't necessarily mean you have to do something, means I'm wondering something. 🎸 means it's rocking, meaning it's a little thing, doesn't really have to be changed, you're good to go. 🚫 means sorry, you gotta fix it. :-)

File: public/styles.css (hunk 8 above)
Comment (TRLevesque) [reply public/styles.css:11]: positioning the brush stroke in the right place

File: public/styles.css (hunk 9 above)
Comment (TRLevesque) [reply public/styles.css]: probably, i think it was a copy paste error.

File: public/styles.css (hunk 10 above)
Comment (TRLevesque) [reply public/styles.css]: probably, i think it was a copy paste error.

File: public/index.html (hunk 1 above)
Comment (TRLevesque) [reply public/index.html:8]: I just copy pasted this from the mark up on rs.com its for all the different site icons. its not real applicable here but i added it for fun.

File: public/index.html (hunk 3 above)
Comment (TRLevesque) [reply public/index.html:35]: gay

**Review by levininja — APPROVED** [review]

**Pull request merged**